io/src/main/scala/fs2/io/io.scala

```diff
@@ -137,7 +147,7 @@ package object io {
         (os: OutputStream, is: InputStream)
       })(
         ois =>
-          Sync[F].delay {
+          blocker.delay {
```

Comment:
FWIW this probably isn't needed; the reason I didn't do it before was to avoid the thread shift. Close in P(I/O)Stream just sets some internal state